shared/i18n/i18n.js

```diff
@@ -0,0 +1,32 @@
+import i18n from "i18next";
+import * as language from "./lang";
+
+const getI18nWithDefaultLang = lang => {
```

Comment:
Wrapped this inside a function so that we can pass the default language to be used.